solr/solrj-zookeeper/src/java/org/apache/solr/common/cloud/CollectionPropertiesZkStateReader.java

```diff
@@ -391,13 +383,7 @@ public void run() {
   private class CacheCleaner implements Runnable {
     @Override
     public void run() {
-      while (!Thread.interrupted()) {
-        try {
-          Thread.sleep(60000);
-        } catch (InterruptedException e) {
-          // Executor shutdown will send us an interrupt
-          break;
-        }
+      if (!closed) {
```

Comment:
don't even need this condition.  The work is not contingent on being closed or not AFAICT.  Even if it did, the pool is going to be closed when it's being shut down so it's okay.

Reply:
Updated.